### DISCUSSION TITLE
‎

### DISCUSSION THREAD

**Discussion by ghost** [VirtualDrivers/Virtual-Display-Driver#176] [Q&A] [answered]

**Comment by bud3699** [comment]
manage Certificates? To be fair, The old cert is invalid now and its not insecure per say as long as the signing files don't get leaked. and as of right now, the only two people who have these files are mike and I

**Reply by ghost** [reply]
Thanks for the obvious answer.

> its not insecure per say as long as the signing files don't get leaked

Well, in the world of security, control is better than trust. No offense, but even big corporations manage to get hacked and their keys leaked if hackers target you directly. Are there any specific reasons to not to include the removal of certificates as part of the uninstall procedure in the readme or at least give an answer here? My question is still not answered.

**Reply by bud3699** [reply, marked as answer]
1) If someone was to care about the security so much to the point of removing certificates, they'd already know how to do so

2) Its understandable that even big corporations get hacked, however big corporations are also a bigger target

3) Videos are being made for uninstalling, installing and configuring the driver

4) Its a root cert, just go into certificate manager, go to root certs then delete it (If you really don't know)
